js/uws/index.js

```diff
@@ -2,13 +2,26 @@
 
 var WebSocketServer = require('uws').Server;
 var wss             = new WebSocketServer({ port: 3334 });
+var cluster         = require('cluster');
+
+if (cluster.isWorker) {
+  process.on('message', function(msg) {
+    wss.broadcast(msg);
+  });
+}
 
 function echo(ws, payload) {
   ws.send(JSON.stringify({type: "echo", payload: payload}));
 }
 
 function broadcast(ws, payload) {
-  wss.broadcast(JSON.stringify({type: "broadcast", payload: payload}));
+  var msg = JSON.stringify({type: "broadcast", payload: payload});
+  wss.broadcast(msg);
```

Comment:
how about we don't broadcast here and then the code above (id !== worker.id) won't be needed.